src/Sylius/Behat/spec/Context/Domain/ProductContextSpec.php

```diff
@@ -45,20 +55,20 @@ function it_implements_context_interface()
     }
 
     function it_deletes_a_product_variant_from_the_repository(
-        ProductVariantRepositoryInterface $productVariantRepository,
+        VariantRepositoryInterface $variantRepository,
         ProductVariantInterface $productVariant
     ) {
-        $productVariantRepository->remove($productVariant)->shouldBeCalled();
+        $variantRepository->remove($productVariant)->shouldBeCalled();
 
         $this->iDeleteTheVariantOfProduct($productVariant);
     }
 
-    function it_tries_to_delete_a_product_variant_that_should_not_be_delted_from_the_repository(
+    function it_tries_to_delete_a_product_variant_that_should_not_be_deleted_from_the_repository(
         SharedStorageInterface $sharedStorage,
-        ProductVariantRepositoryInterface $productVariantRepository,
+        VariantRepositoryInterface $variantRepository,
```

Comment:
That is wrong since in the constructor we require `ProductVariantRepositoryInterface` and the variable name should reflect the fact that it is a `productVariantRepository`